### ISSUE TITLE
Read-Only datepicker.

### ISSUE THREAD

**Issue by InKairo** [mymth/vanillajs-datepicker#113]
Can we add an option to config the datepicker so that if it's true, it shows the datepicker with the selected date or do not show the datepicker at all?

The main reason I want this is because I have an input that holds the datepicker, but there are several occasions where I want the date not to be changed, only shown.

So I though that maybe reading the value of the readonly input and hiding the datepicker in a show event could work, but I see this as a hacky way of preventing something to show in the first place. 

So having an option to make the whole datepicker "readonly" would benefit not only performance, but avoid potential flickers on screen (due to fast show and hide) and also make inline datepickers able to show a date but not allow to change it. 

I'm very sorry if this already exist on some way, but I read the whole documentation several times and didn't find something that could fit this need. Thanks in advance! I think this is the best picker all around!

**Comment by InKairo** [comment]
so, to show visually what I think would be awesome and simple, is to:
if option
   read-only : true
then
   hide prev and next buttons
   change background
   disable onclick events over the datepicker
   
The background could be handled with an array on the read-only option like:
read-only: [true, "#dbdbdb"]

![image](https://user-images.githubusercontent.com/10782313/169637839-a6e01bd2-46d0-4165-a19f-84000f51827c.png)

**Comment by mymth** [comment]
Is this like what you are trying to make?
https://codepen.io/mymth/pen/qBxXEKV

By the way, adding an option not to show datepicker when input has `readonly` attribute is planned.
(see the second last paragraph of https://github.com/mymth/vanillajs-datepicker/pull/98#issuecomment-1031125652)

I think the specs in your second post doesn't necessarily meet most people's preferences and some other people probably prefer a bit different way to make the picker read-only. So I'm reluctant to add such an option and would like users to make their own version of read-only datepicker themselves because, as you can see in the above demo, it can be done without touching the library's source code.

**Comment by InKairo** [comment]
Yes! that is exactly what I need! I'll check it out because I still develop in plain js/jquery

Well. It's true that you can do everything without touching the library's source code. I think that the readonly attribute is just enough for most of people needs. Thanks!
